message/tests.py

```diff
@@ -14,93 +15,126 @@
 TEST_BALL_GAME = 'Basketball'
 
 
+@pytest.fixture
+def message(player, game_event):
+    message = Message.objects.create(
+        user_id=player,
+        game_event_id=game_event,
+        time_sent=TEST_TIME,
+        text="test message"
+    )
+    return message
+
+
 @pytest.mark.django_db
 class TestMessageModel:
 
-    @pytest.fixture
-    def user(self):
-        return get_user_model().objects.create_user(
-            username='testuser',
-            password='testpass'
-        )
-
-    @pytest.fixture
-    def saved_player(self, user):
-        player = Player.objects.create(
-            user=user,
-            birth_date='1990-01-01',
-            favorite_ball_game=BallGame.Soccer
-        )
-        return player
-
-    @pytest.fixture
-    def saved_game_event(self, court):
-        return GameEvent.objects.create(id=TEST_ID, time=TEST_TIME, level_of_game=TEST_LEVEL,
-                                        min_number_of_players=TEST_MIN, max_number_of_players=TEST_MAX,
-                                        court=court, ball_game=TEST_BALL_GAME)
-
-    @pytest.fixture
-    def saved_message(self, saved_player, saved_game_event):
-        message = Message.objects.create(
-            user_id=saved_player,
-            game_event_id=saved_game_event,
-            time_sent=TEST_TIME,
-            text="test message"
-        )
-        return message
```

Comment:
Are those fixture changes related to this PR?

Reply:
> Are those fixture changes related to this PR?

Those fixtures are inside `class TestMessageModel`. If I not put those fixtures outside of this class,
my tests will not recognize those fixtures and I will duplicates those fixtures. But according to #102 it isn't necessary right now.
I will wait to merge #102 PR and fix it. About the other changes - you're right, I will open an issue about it and open ne PR for it.

Reply:
> Are those fixture changes related to this PR?

About the position of those fixture I think yes, because those fixtures are inside the class `TestMessageModel`, and my new tests aren't recognize those fixtures because of it. About the renaming, you're right and I open an issue about it #106